lib/std/array_list.zig

```diff
@@ -6,6 +6,20 @@ const mem = std.mem;
 const math = std.math;
 const Allocator = mem.Allocator;
 
+/// Shared between managed and unmanaged versions of ArrayList. Called
+/// when memory growth is necessary. Returns a capacity larger than minimum
+/// that is better according to our growth policy.
+/// In future versions of (Unmanaged)ArrayList, we should consider taking
+/// the growth policy as parameter.
```

Comment:
Wishful thinking does not belong in doc comments. Please delete this and file an issue or pull request instead.